### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by yuchaoran2011**
As part of support work on Friday, I fixed a bunch of minor things in this PR:
1. https://github.com/lightbend/cloudflow/issues/37
2. https://github.com/lightbend/cloudflow/issues/67
3. Some improvements to style of Go code in the CLI, as suggested by my IDE (GoLand). Specifically, error message should start with lower-case letter and shouldn't end with punctuation.

### CODE REVIEW

File: installer/create-cluster-gke.sh [hunk 1]
```diff
@@ -40,15 +40,18 @@ if [ "$gcloudRegion" == "" ]
...
-
-CLUSTER_VERSION=`gcloud container get-server-config --format json | jq -r .defaultClusterVersion`
+if [ -z "$CLUSTER_VERSION" ]
+  then
+    # https://cloud.google.com/kubernetes-engine/versioning-and-upgrades#versions_available_for_new_cluster_masters
+    CLUSTER_VERSION=`gcloud container get-server-config --format json | jq -r .defaultClusterVersion`
+    echo "No cluster version specified. Using the default: $CLUSTER_VERSION"
+  else
+    echo "Cluster version: $CLUSTER_VERSION"
+fi
```
Comment (agemooij) [inline installer/create-cluster-gke.sh]: Nice!

File: kubectl-cloudflow/cmd/configure.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func (c *configureApplicationCMD) configureImpl(cmd *cobra.Command, args []strin
 func validateConfigureCMDArgs(cmd *cobra.Command, args []string) error {
 
 	if len(args) < 1 {
-		return fmt.Errorf("You need to specify a Cloudflow application name and the required configuration key/value pair(s)")
+		return fmt.Errorf("you need to specify a Cloudflow application name and the required configuration key/value pair(s)")
```
Comment (agemooij) [inline kubectl-cloudflow/cmd/configure.go]: Shoudn't this start with a capital?

File: kubectl-cloudflow/cmd/deploy.go [hunk 3]
```diff
@@ -387,7 +387,7 @@ func createOrUpdateCloudflowApplication(cloudflowApplicationClient *k8s.Cloudflo
 func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
 
 	if len(args) < 1 || args[0] == "" {
-		return fmt.Errorf("Please specify the full path to the Docker image containing the application. For example: 'docker-registry.mydomain.com/cloudflow/awesome-app:37-172e856'")
+		return fmt.Errorf("please specify the full path to the Docker image containing the application. For example: 'docker-registry.mydomain.com/cloudflow/awesome-app:37-172e856'")
```
Comment (agemooij) [inline kubectl-cloudflow/cmd/deploy.go]: This is a new sentence, right?

**Comment by olofwalker** [comment]
@agemooij error messages in go should not be capitalized, doing so produces a warning.

File: kubectl-cloudflow/cmd/deploy.go (hunk 3 above)
Comment (debasishg) [reply kubectl-cloudflow/cmd/deploy.go]: @agemooij Please take a look at https://github.com/golang/go/wiki/CodeReviewComments#error-strings .. golang encourages this convention :-)

Comment (agemooij) [reply kubectl-cloudflow/cmd/deploy.go]: OK, I'd then like for us to check that the resulting log messages actually make sense, e.g. don't only change the initial capital but also check where the error message actually gets logged and then make sure the resulting sentence makes sense.

**Pull request merged**